src/reactive-membrane.ts

```diff
@@ -68,6 +68,24 @@ const defaultValueMutated: ReactiveMembraneMutationCallback = (obj: any, key: Pr
 };
 const defaultValueDistortion: ReactiveMembraneDistortionCallback = (value: any) => value;
 
+export function wrapDescriptor(membrane: ReactiveMembrane, descriptor: PropertyDescriptor, getValue: (membrane: ReactiveMembrane, originalValue: any) => any): PropertyDescriptor {
+    const { set, get } = descriptor;
+    if ('value' in descriptor) {
+        descriptor.value = getValue(membrane, descriptor.value);
+    }
+    if (!isUndefined(get)) {
```

Comment:
`value` mutually exclusive with `get` and `set`. So we can create 2 branches, so you don't need to run the `get` and `set` branch if `value` is present.

```js
if (descriptor.hasOwnProperty('value')) {
	// ...
} else {
	if (!isUndefined(get)) {
		//...
	}
	if (!isUndefined(set)) {
		//...
	}
}
```